src/views/Reticket.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { size } from 'lodash';
+import * as need from 'lib/need';
+import ob from 'urbit-ob';
+import { Just, Nothing } from 'folktale/maybe';
+
+import { usePointCursor } from 'store/pointCursor';
+
+import View from 'components/View';
+import MiniBackButton from 'components/MiniBackButton';
+import { LocalRouterProvider } from 'lib/LocalRouter';
+import Crumbs from 'components/Crumbs';
+
+import useRouter from 'lib/useRouter';
+
+import { useHistory } from 'store/history';
+
+import Confirm from './Reticket/Confirm';
+import Download from './Reticket/Download';
+import Verify from './Reticket/Verify';
+import DoReticket from './Reticket/DoReticket';
+
+const STEP_NAMES = {
```

Comment:
let's just call them `NAMES` and `VIEWS` since they're file-local, which is what I've done with the other nested routers